README.md

```diff
@@ -27,9 +27,9 @@ Want to help with development? [Join our discord!](https://discord.gg/nZKAzzE5bX
 
 # Other Notes
 
-While QuACS and the developers are huge fans of [RCOS](https://rcos.io/), we are not affiliated with RCOS at this time. If you want to help out with development, join our [Discord server](https://discord.gg/EyGZTAP).
+QuACS is a proud member of the [Rensselaer Center for Open Source](https://rcos.io/)
 
-We are also not affiliated with YACS. The work they have done has been invaluable to the RPI student body and while our codebase is different, much of our design was inspired by YACS. We however also support competition and believe we provide a number of features that YACS does not. We are hoping you try us out when you register for classes next semester.
+QuACS is in no way affiliated with YACS. The work they have done has been invaluable to the RPI student body and while our codebase is different, much of our design was inspired by YACS. We however also support competition and believe we provide a number of features that YACS does not. We are hoping you try us out when you register for classes next semester.
```

Comment:
I think this should be reworded to `is not affiliated` instead of `is in no way`.  The latter is too harsh in my opinion.